dependency-check-suppressions.xml

```diff
@@ -89,6 +89,11 @@
         <cve>CVE-2020-15250</cve>
     </suppress>
 
+    <suppress>
+        <notes>This suppress specific cve for Jetty Library which is a transitive dependency</notes>
```

Comment:
transitive doesn't matter you still have it, it should be patched if available or at a minimum needs an expiry date

Reply:
@timja This transitive dependency also has CVE issues and hence this supression is added